### ISSUE TITLE
Java Script Error

### ISSUE THREAD

**Issue by Prakash71322** [ransome1/sleek#404]
JavaScript Error TypeError Cannot Read the properties of undefined (hide) @ Browser Window

**Comment by bike-bill** [comment]
It looks like this is a duplicate of #389

**Comment by ransome1** [comment]
@bike-bill I agree, this is at least related. @Prakash71322 hope you don't mind if I close here. If this isn't solved with the fix in the other issue, we can reopen here.
